stable/prometheus/README.md

```diff
@@ -140,6 +140,7 @@ Parameter | Description | Default
 `server.service.externalIPs` | Prometheus server service external IP addresses | `[]`
 `server.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `server.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`server.service.nodePort` | Port to be used as the service NodePort (ignored if `service.service.type` is not `NodePort`) | `None`
```

Comment:
ignored if `**server**.service.type`...

And let's set the default value to `0`